apps/api/src/schema/websoc.ts

```diff
@@ -118,30 +136,69 @@ export const websocQuerySchema = z.object({
       }
       return parsedNums;
     }),
-  instructorName: z.string().optional(),
-  days: daysSchema.optional(),
-  building: z.string().optional(),
-  room: z.string().optional(),
+  instructorName: z.string().optional().openapi({
+    description: "The instructor's name for the course",
+    example: "Anteater, P.",
+  }),
+  days: daysSchema.optional().openapi({}),
+  building: z.string().optional().openapi({
+    description: "The building where the course is held",
+    example: "ALP",
+  }),
+  room: z.string().optional().openapi({
+    description: "The room number where the course is held",
+    example: "1100",
+  }),
   division: z
     .enum(courseLevels)
     .or(z.literal("ANY"))
     .optional()
+    .openapi({
+      description: "Filter courses by course level",
+      example: "LowerDiv",
+    })
     .transform((x) => (x === "ANY" ? undefined : x)),
   sectionType: z
     .union([z.enum(anyArray), z.enum(websocSectionTypes)])
     .optional()
+    .openapi({
+      description: "Filter courses by section type",
```

Comment:
Expanding on the comment extending throughout this review: A section type describes a section; a course as a whole is not a lecture or recitation. In fact, a course can have sections of only one type or of multiple. Then it doesn't make sense to claim you can "filter courses by section type". Reword accordingly.